tests/app-layer-template/test.yaml

```diff
@@ -5,6 +5,7 @@ args:
 
 checks:
 - filter:
+    lt-version: 7
```

Comment:
It looks to me that we have twice the same test for `lt-version: 7` and `min-version: 7`
What am I missing ?

Reply:
`template.request` and `template.response` differ.